src/lib.rs

```diff
@@ -398,6 +398,8 @@ pub struct ThinVec<T> {
     boo: PhantomData<T>,
 }
 
+unsafe impl<T> Sync for ThinVec<T> {}
+unsafe impl<T> Send for ThinVec<T> {}
```

Comment:
Both of these need to require that T has the same property, just like Vec does:

https://doc.rust-lang.org/std/vec/struct.Vec.html#impl-Send

Reply:
Thanks for pointing this out!

I've fixed this.